company.el

```diff
@@ -1448,6 +1448,55 @@ end of the match."
                                 candidate
                                 selected))
 
+(defvar company-unicode-icons-mapping
```

Comment:
Maybe call it `company-text-icons-mapping`? Greek chars are just one choice.

If someone installs FontAwesome, I think they'll be able to use it here by customizing this mapping (right?), even though that font's "characters" don't exactly correspond to Unicode tables.

Reply:
Done.